File: coach-ticket-booking-api/Models/Route.cs
using coach_ticket_booking_api.Enums;
using System.ComponentModel.DataAnnotations;

namespace coach_ticket_booking_api.Models
{
    public class Route
    {
        [Key]
        public Guid Id { get; set; }
        public Guid FromOfficeID { get; set; }
        public Office FromOffice { get; set; }
        public Guid ToOfficeID { get; set; }
        public Office ToOffice { get; set; }
        public DateTime CreateDate { get; set; } = DateTime.Now;
        public RouteStatus Status { get; set; }
        public virtual ICollection<TimeToOffice> OfficesInRoute { get; set; } = new HashSet<TimeToOffice>();
    }
}

File: coach-ticket-booking/src/app/models/route.ts
export interface Route {
  id: string;
  fromOfficeID: string;
  toOfficeID: string;
  createDate: string;
  status: number;
  fromOffice: {
    id: string;
    name: string | null;
    address: string | null;
    townName: string | null;
  };
  toOffice: {
    id: string;
    name: string | null;
    address: string | null;
    townName: string | null;
  };
}

export interface RoutesParams{
    pageNumber:number;
    pageSize:number;
}